app/package.json

```diff
@@ -69,7 +69,7 @@
     "vue": "^2.6.10",
     "vue-knob-control": "^1.6.0",
     "vue-router": "^3.0.7",
-    "vuetify": "^1.5.16",
+    "vuetify": "^2.0.0-beta.5",
```

Comment:
Still more errors in console for vuetify. pls resolve them

Reply:
> Still more errors in console for vuetify. pls resolve them
Done.

Reply:
> Still more errors in console for vuetify. pls resolve them

Done